test_itsm.py

```diff
@@ -815,3 +817,208 @@ def test_create_comment(client):
 
     # Should be the same as what we got in our POST response
     assert post_response == comment
+
+
+def test_attachments_list(client):
+    # Get the url to the attachments list resource, then get the list of attachments that are
+    # currently stored in the database.
+    attachments_url = _get_root_links(client)['attachments']
+    attachments     = client.get(attachments_url, **JSON_HDRS_READ).get_json()
+
+    # The test jpeg image attachment's base64 encoding is quite large - so here we use the
+    # base64 library to create the encoding and store it in a variable instead of having the
+    # literal string in the assert data.
+    with open("test_data/mt-fuji.jpeg", "rb") as image_attachment:
+        encoded_image_attachment = base64.b64encode(image_attachment.read()).decode()
```

Comment:
It seems this variable isn't used in this test?

Reply:
Right, an artifact from some previous testing. Removed.